### DISCUSSION TITLE
print bluetooth symbol on st7789 lcd

### DISCUSSION THREAD

**Discussion by ghost** [Bodmer/TFT_eSPI#2586] [General]
HI team,
I am new to this Arduino platform. i need to print the Bluetooth symbol on 1.3 inch st7789 lcd. how to print that one.

please provide some guidance or help to achieve this one.

thanks

**Comment by Bodmer** [comment]
See this example for a possible approach:
https://github.com/Bodmer/TFT_eSPI/blob/master/examples/Generic/TFT_Flash_Bitmap/TFT_Flash_Bitmap.ino

**Comment by Bodmer** [comment]
The easy way to convert a bmp format image to a C array for inclusion in the above sketch is to use the tool here:

https://notisrac.github.io/FileToCArray/

Example settings to generate the right file format:

![image](https://user-images.githubusercontent.com/15824805/233131142-5ff58097-8346-48d1-8cfc-ecdaea54d415.png)

The line to display that particular array would then be:

tft.pushImage(0, 0, TIGER_WIDTH, TIGER_HEIGHT, Tiger);

**Reply by Bodmer** [reply]
Here the array has been pasted into the sketch:
![image](https://user-images.githubusercontent.com/15824805/233132236-8677c7ab-92a6-4d5f-9eb2-57eba3a9f284.png)

In this example add the following line to the sketch:
#include "Tiger.h"

Draw with:
tft.pushImage(0, 0, TIGER_WIDTH, TIGER_HEIGHT, Tiger);
